2024-12-12 18:16:49.950191+01:00 database: close
org.h2.message.DbException: IO Exception: "Closing"
IO Exception: "Closing" [90028-232]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.close(Store.java:374)
	at org.h2.engine.Database.closeOpenFilesAndUnlock(Database.java:1292)
	at org.h2.engine.Database.closeImpl(Database.java:1254)
	at org.h2.engine.Database.close(Database.java:1173)
	at org.h2.engine.Database.onShutdown(Database.java:1158)
	at org.h2.engine.OnExitDatabaseCloser.onShutdown(OnExitDatabaseCloser.java:85)
	at org.h2.engine.OnExitDatabaseCloser.run(OnExitDatabaseCloser.java:114)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "Closing"
IO Exception: "Closing" [90028-232]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 8 more
Caused by: org.h2.mvstore.MVStoreException: java.lang.AssertionError [2.3.232/3]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.MVStore.executeFilestoreOperation(MVStore.java:926)
	at org.h2.mvstore.RandomAccessStore.compactMoveChunks(RandomAccessStore.java:463)
	at org.h2.mvstore.RandomAccessStore.compactStore(RandomAccessStore.java:443)
	at org.h2.mvstore.FileStore.compactStore(FileStore.java:890)
	at org.h2.mvstore.FileStore.stop(FileStore.java:275)
	at org.h2.mvstore.MVStore.closeStore(MVStore.java:693)
	at org.h2.mvstore.MVStore.close(MVStore.java:643)
	at org.h2.mvstore.db.Store.close(Store.java:364)
	... 6 more
Caused by: java.lang.AssertionError
	at org.h2.mvstore.RandomAccessStore.moveChunkInside(RandomAccessStore.java:630)
	at org.h2.mvstore.RandomAccessStore.compactMoveChunks(RandomAccessStore.java:569)
	at org.h2.mvstore.RandomAccessStore.compactMoveChunks(RandomAccessStore.java:481)
	at org.h2.mvstore.RandomAccessStore.lambda$compactMoveChunks$1(RandomAccessStore.java:468)
	at org.h2.mvstore.FileStore.executeFileStoreOperation(FileStore.java:1893)
	at org.h2.mvstore.MVStore.executeFilestoreOperation(MVStore.java:922)
	... 13 more
2024-12-12 19:55:47.496771+01:00 database: close
org.h2.message.DbException: IO Exception: "Closing"
IO Exception: "Closing" [90028-232]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.close(Store.java:374)
	at org.h2.engine.Database.closeOpenFilesAndUnlock(Database.java:1292)
	at org.h2.engine.Database.closeImpl(Database.java:1254)
	at org.h2.engine.Database.close(Database.java:1173)
	at org.h2.engine.Database.onShutdown(Database.java:1158)
	at org.h2.engine.OnExitDatabaseCloser.onShutdown(OnExitDatabaseCloser.java:85)
	at org.h2.engine.OnExitDatabaseCloser.run(OnExitDatabaseCloser.java:114)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "Closing"
IO Exception: "Closing" [90028-232]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 8 more
Caused by: org.h2.mvstore.MVStoreException: java.lang.AssertionError [2.3.232/3]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.MVStore.executeFilestoreOperation(MVStore.java:926)
	at org.h2.mvstore.RandomAccessStore.compactMoveChunks(RandomAccessStore.java:463)
	at org.h2.mvstore.RandomAccessStore.compactStore(RandomAccessStore.java:443)
	at org.h2.mvstore.FileStore.compactStore(FileStore.java:890)
	at org.h2.mvstore.FileStore.stop(FileStore.java:275)
	at org.h2.mvstore.MVStore.closeStore(MVStore.java:693)
	at org.h2.mvstore.MVStore.close(MVStore.java:643)
	at org.h2.mvstore.db.Store.close(Store.java:364)
	... 6 more
Caused by: java.lang.AssertionError
	at org.h2.mvstore.RandomAccessStore.moveChunkInside(RandomAccessStore.java:630)
	at org.h2.mvstore.RandomAccessStore.compactMoveChunks(RandomAccessStore.java:569)
	at org.h2.mvstore.RandomAccessStore.compactMoveChunks(RandomAccessStore.java:481)
	at org.h2.mvstore.RandomAccessStore.lambda$compactMoveChunks$1(RandomAccessStore.java:468)
	at org.h2.mvstore.FileStore.executeFileStoreOperation(FileStore.java:1893)
	at org.h2.mvstore.MVStore.executeFilestoreOperation(MVStore.java:922)
	... 13 more
2024-12-13 18:32:51.664443+01:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabla "USUARIOS" no encontrada
Table "USUARIOS" not found; SQL statement:
INSERT INTO USUARIOS (ID, PASSWORD, ROL, USERNAME, MEDICO_ID, PACIENTE_ID) 
VALUES (2, 'securepassword', 'Medico', 'carlos', 2, NULL) [42102-232]
2024-12-16 17:06:51.932468+01:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "[*]¨SELECT * FROM MEDICO"
Syntax error in SQL statement "[*]¨SELECT * FROM MEDICO"; SQL statement:
¨SELECT * FROM MEDICO [42000-232]
2024-12-16 17:07:02.970245+01:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "[*]¨SELECT * FROM MEDICO"
Syntax error in SQL statement "[*]¨SELECT * FROM MEDICO"; SQL statement:
¨SELECT * FROM MEDICO [42000-232]
2024-12-16 17:16:19.357732+01:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "INSERT INTO PACIENTE (ID, APELLIDO, EMAIL, NOMBRE, MEDICO_ID, CONDICION_MEDICA, FRECUENCIA_CARDIACA, FRECUENCIA_RESPIRATORIA, NIVEL_ACTIVIDAD_FISICA, NIVEL_GLUCOSA, PASSWORD, PRESION_ARTERIAL, SATURACIONO2, USERNAME) VALUES\000d\000a-- Pacientes con Diabetes\000d\000a(1, 'Perez', 'juanperez@example.com', 'Juan Perez', 1, 'Diabetes', NULL, NULL, 7, 120.5, 'securepassword', NULL, NULL, 'juanperez'),\000d\000a(2, 'Garcia', 'mariagarcia@example.com', 'Maria Garcia', 2, 'Diabetes', NULL, NULL, 8, 110.3, 'securepassword', NULL, NULL, 'mariagarcia'),\000d\000a\000d\000a-- Pacientes con Hipertensión\000d\000a(3, 'Fernandez', 'carlosfernandez@example.com', 'Carlos Fernandez', 3, 'Hipertensión', 80, NULL, NULL, NULL, 'securepassword', '130/85', NULL, 'carlosf'),\000d\000a(4, 'Sanchez', 'anitasanchez@example.com', 'Ana Sanchez', 4, 'Hipertensión', 75, NULL, NULL, NULL, 'securepassword', '140/90', NULL, 'anitas'),\000d\000a\000d\000a-- Pacientes con EPOC\000d\000a(5, 'Lopez', 'manuel.lopez@example.com', 'Manuel Lopez', 1, 'EPOC', NULL, 22, NULL, NULL, 'securepassword', NULL, 88.5, 'manuellopez'),\000d\000a(6, 'Martinez', 'susana.martinez@example.com', 'Susana Martinez', 2, 'EPOC', NULL, 25, NULL, NULL, 'securepassword', NULL, 90.2, 'susanam'),\000d\000a-- Paciente con Hipertension y EPOC\000d\000a(7, 'Rojas', 'javier.rojas@example.com', 'Javier Rojas', 3, 'Hipertension EPOC NULL'[*]NULL)"; se esperaba "[, ., ::, AT, FORMAT, *, /, %, +, -, ||, NOT, IS, ILIKE, REGEXP, AND, OR, ,, )"
Syntax error in SQL statement "INSERT INTO PACIENTE (ID, APELLIDO, EMAIL, NOMBRE, MEDICO_ID, CONDICION_MEDICA, FRECUENCIA_CARDIACA, FRECUENCIA_RESPIRATORIA, NIVEL_ACTIVIDAD_FISICA, NIVEL_GLUCOSA, PASSWORD, PRESION_ARTERIAL, SATURACIONO2, USERNAME) VALUES\000d\000a-- Pacientes con Diabetes\000d\000a(1, 'Perez', 'juanperez@example.com', 'Juan Perez', 1, 'Diabetes', NULL, NULL, 7, 120.5, 'securepassword', NULL, NULL, 'juanperez'),\000d\000a(2, 'Garcia', 'mariagarcia@example.com', 'Maria Garcia', 2, 'Diabetes', NULL, NULL, 8, 110.3, 'securepassword', NULL, NULL, 'mariagarcia'),\000d\000a\000d\000a-- Pacientes con Hipertensión\000d\000a(3, 'Fernandez', 'carlosfernandez@example.com', 'Carlos Fernandez', 3, 'Hipertensión', 80, NULL, NULL, NULL, 'securepassword', '130/85', NULL, 'carlosf'),\000d\000a(4, 'Sanchez', 'anitasanchez@example.com', 'Ana Sanchez', 4, 'Hipertensión', 75, NULL, NULL, NULL, 'securepassword', '140/90', NULL, 'anitas'),\000d\000a\000d\000a-- Pacientes con EPOC\000d\000a(5, 'Lopez', 'manuel.lopez@example.com', 'Manuel Lopez', 1, 'EPOC', NULL, 22, NULL, NULL, 'securepassword', NULL, 88.5, 'manuellopez'),\000d\000a(6, 'Martinez', 'susana.martinez@example.com', 'Susana Martinez', 2, 'EPOC', NULL, 25, NULL, NULL, 'securepassword', NULL, 90.2, 'susanam'),\000d\000a-- Paciente con Hipertension y EPOC\000d\000a(7, 'Rojas', 'javier.rojas@example.com', 'Javier Rojas', 3, 'Hipertension EPOC NULL'[*]NULL)"; expected "[, ., ::, AT, FORMAT, *, /, %, +, -, ||, NOT, IS, ILIKE, REGEXP, AND, OR, ,, )"; SQL statement:
INSERT INTO PACIENTE (ID, APELLIDO, EMAIL, NOMBRE, MEDICO_ID, CONDICION_MEDICA, FRECUENCIA_CARDIACA, FRECUENCIA_RESPIRATORIA, NIVEL_ACTIVIDAD_FISICA, NIVEL_GLUCOSA, PASSWORD, PRESION_ARTERIAL, SATURACIONO2, USERNAME) VALUES
-- Pacientes con Diabetes
(1, 'Perez', 'juanperez@example.com', 'Juan Perez', 1, 'Diabetes', NULL, NULL, 7, 120.5, 'securepassword', NULL, NULL, 'juanperez'),
(2, 'Garcia', 'mariagarcia@example.com', 'Maria Garcia', 2, 'Diabetes', NULL, NULL, 8, 110.3, 'securepassword', NULL, NULL, 'mariagarcia'),

-- Pacientes con Hipertensión
(3, 'Fernandez', 'carlosfernandez@example.com', 'Carlos Fernandez', 3, 'Hipertensión', 80, NULL, NULL, NULL, 'securepassword', '130/85', NULL, 'carlosf'),
(4, 'Sanchez', 'anitasanchez@example.com', 'Ana Sanchez', 4, 'Hipertensión', 75, NULL, NULL, NULL, 'securepassword', '140/90', NULL, 'anitas'),

-- Pacientes con EPOC
(5, 'Lopez', 'manuel.lopez@example.com', 'Manuel Lopez', 1, 'EPOC', NULL, 22, NULL, NULL, 'securepassword', NULL, 88.5, 'manuellopez'),
(6, 'Martinez', 'susana.martinez@example.com', 'Susana Martinez', 2, 'EPOC', NULL, 25, NULL, NULL, 'securepassword', NULL, 90.2, 'susanam'),
-- Paciente con Hipertension y EPOC
(7, 'Rojas', 'javier.rojas@example.com', 'Javier Rojas', 3, 'Hipertension EPOC NULL'NULL) [42001-232]
2024-12-16 17:16:46.886023+01:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "INSERT INTO PACIENTE (ID, APELLIDO, EMAIL, NOMBRE, MEDICO_ID, CONDICION_MEDICA, FRECUENCIA_CARDIACA, FRECUENCIA_RESPIRATORIA, NIVEL_ACTIVIDAD_FISICA, NIVEL_GLUCOSA, PASSWORD, PRESION_ARTERIAL, SATURACIONO2, USERNAME) VALUES\000d\000a(1, 'Perez', 'juanperez@example.com', 'Juan Perez', 1, 'Diabetes', NULL, NULL, 7, 120.5, 'securepassword', NULL, NULL, 'juanperez'),\000d\000a(2, 'Garcia', 'mariagarcia@example.com', 'Maria Garcia', 2, 'Diabetes', NULL, NULL, 8, 110.3, 'securepassword', NULL, NULL, 'mariagarcia'),[*]"; se esperaba "ROW (, (, DEFAULT, INTERSECTS (, NOT, EXISTS, UNIQUE, INTERSECTS"
Syntax error in SQL statement "INSERT INTO PACIENTE (ID, APELLIDO, EMAIL, NOMBRE, MEDICO_ID, CONDICION_MEDICA, FRECUENCIA_CARDIACA, FRECUENCIA_RESPIRATORIA, NIVEL_ACTIVIDAD_FISICA, NIVEL_GLUCOSA, PASSWORD, PRESION_ARTERIAL, SATURACIONO2, USERNAME) VALUES\000d\000a(1, 'Perez', 'juanperez@example.com', 'Juan Perez', 1, 'Diabetes', NULL, NULL, 7, 120.5, 'securepassword', NULL, NULL, 'juanperez'),\000d\000a(2, 'Garcia', 'mariagarcia@example.com', 'Maria Garcia', 2, 'Diabetes', NULL, NULL, 8, 110.3, 'securepassword', NULL, NULL, 'mariagarcia'),[*]"; expected "ROW (, (, DEFAULT, INTERSECTS (, NOT, EXISTS, UNIQUE, INTERSECTS"; SQL statement:
INSERT INTO PACIENTE (ID, APELLIDO, EMAIL, NOMBRE, MEDICO_ID, CONDICION_MEDICA, FRECUENCIA_CARDIACA, FRECUENCIA_RESPIRATORIA, NIVEL_ACTIVIDAD_FISICA, NIVEL_GLUCOSA, PASSWORD, PRESION_ARTERIAL, SATURACIONO2, USERNAME) VALUES
(1, 'Perez', 'juanperez@example.com', 'Juan Perez', 1, 'Diabetes', NULL, NULL, 7, 120.5, 'securepassword', NULL, NULL, 'juanperez'),
(2, 'Garcia', 'mariagarcia@example.com', 'Maria Garcia', 2, 'Diabetes', NULL, NULL, 8, 110.3, 'securepassword', NULL, NULL, 'mariagarcia'), [42001-232]
2024-12-16 17:18:43.060588+01:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "INSERT INTO PACIENTE (ID, APELLIDO, EMAIL, NOMBRE, MEDICO_ID, CONDICION_MEDICA, FRECUENCIA_CARDIACA, FRECUENCIA_RESPIRATORIA, NIVEL_ACTIVIDAD_FISICA, NIVEL_GLUCOSA, PASSWORD, PRESION_ARTERIAL, SATURACIONO2, USERNAME) VALUES(3, 'Fernandez', 'carlosfernandez@example.com', 'Carlos Fernandez', 3, 'Hipertensión', 80, NULL, NULL, NULL, 'securepassword', '130/85', NULL, 'carlosf'),\000d\000a(4, 'Sanchez', 'anitasanchez@example.com', 'Ana Sanchez', 4, 'Hipertensión', 75, NULL, NULL, NULL, 'securepassword', '140/90', NULL, 'anitas'),[*]"; se esperaba "ROW (, (, DEFAULT, INTERSECTS (, NOT, EXISTS, UNIQUE, INTERSECTS"
Syntax error in SQL statement "INSERT INTO PACIENTE (ID, APELLIDO, EMAIL, NOMBRE, MEDICO_ID, CONDICION_MEDICA, FRECUENCIA_CARDIACA, FRECUENCIA_RESPIRATORIA, NIVEL_ACTIVIDAD_FISICA, NIVEL_GLUCOSA, PASSWORD, PRESION_ARTERIAL, SATURACIONO2, USERNAME) VALUES(3, 'Fernandez', 'carlosfernandez@example.com', 'Carlos Fernandez', 3, 'Hipertensión', 80, NULL, NULL, NULL, 'securepassword', '130/85', NULL, 'carlosf'),\000d\000a(4, 'Sanchez', 'anitasanchez@example.com', 'Ana Sanchez', 4, 'Hipertensión', 75, NULL, NULL, NULL, 'securepassword', '140/90', NULL, 'anitas'),[*]"; expected "ROW (, (, DEFAULT, INTERSECTS (, NOT, EXISTS, UNIQUE, INTERSECTS"; SQL statement:
INSERT INTO PACIENTE (ID, APELLIDO, EMAIL, NOMBRE, MEDICO_ID, CONDICION_MEDICA, FRECUENCIA_CARDIACA, FRECUENCIA_RESPIRATORIA, NIVEL_ACTIVIDAD_FISICA, NIVEL_GLUCOSA, PASSWORD, PRESION_ARTERIAL, SATURACIONO2, USERNAME) VALUES(3, 'Fernandez', 'carlosfernandez@example.com', 'Carlos Fernandez', 3, 'Hipertensión', 80, NULL, NULL, NULL, 'securepassword', '130/85', NULL, 'carlosf'),
(4, 'Sanchez', 'anitasanchez@example.com', 'Ana Sanchez', 4, 'Hipertensión', 75, NULL, NULL, NULL, 'securepassword', '140/90', NULL, 'anitas'), [42001-232]
2024-12-16 23:43:45.794307+01:00 jdbc[15]: null
org.h2.message.DbException: La conexión no fue cerrada por la aplicación y esta siendo limpiada (garbage collected)
The connection was not closed by the application and is garbage collected [90018-232]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:217)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:134)
	at org.h2.Driver.connect(Driver.java:59)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:137)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:360)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:202)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:461)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:550)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:98)
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:111)
	at org.springframework.boot.autoconfigure.h2.H2ConsoleAutoConfiguration.getConnectionUrl(H2ConsoleAutoConfiguration.java:94)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.stream.SortedOps$RefSortingSink.end(SortedOps.java:395)
	at java.base/java.util.stream.Sink$ChainedReference.end(Sink.java:261)
	at java.base/java.util.stream.Sink$ChainedReference.end(Sink.java:261)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:510)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:575)
	at java.base/java.util.stream.AbstractPipeline.evaluateToArrayNode(AbstractPipeline.java:260)
	at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:616)
	at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:622)
	at java.base/java.util.stream.ReferencePipeline.toList(ReferencePipeline.java:627)
	at org.springframework.boot.autoconfigure.h2.H2ConsoleAutoConfiguration.logDataSources(H2ConsoleAutoConfiguration.java:86)
	at org.springframework.boot.autoconfigure.h2.H2ConsoleAutoConfiguration.lambda$h2Console$0(H2ConsoleAutoConfiguration.java:69)
	at org.springframework.boot.autoconfigure.h2.H2ConsoleAutoConfiguration.withThreadContextClassLoader(H2ConsoleAutoConfiguration.java:78)
	at org.springframework.boot.autoconfigure.h2.H2ConsoleAutoConfiguration.h2Console(H2ConsoleAutoConfiguration.java:69)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.lambda$instantiate$0(SimpleInstantiationStrategy.java:171)
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiateWithFactoryMethod(SimpleInstantiationStrategy.java:88)
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:168)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:653)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:645)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1351)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1181)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:563)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:523)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:336)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:288)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:334)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:204)
	at org.springframework.boot.web.servlet.ServletContextInitializerBeans.getOrderedBeansOfType(ServletContextInitializerBeans.java:211)
	at org.springframework.boot.web.servlet.ServletContextInitializerBeans.getOrderedBeansOfType(ServletContextInitializerBeans.java:202)
	at org.springframework.boot.web.servlet.ServletContextInitializerBeans.addServletContextInitializerBeans(ServletContextInitializerBeans.java:97)
	at org.springframework.boot.web.servlet.ServletContextInitializerBeans.<init>(ServletContextInitializerBeans.java:86)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.getServletContextInitializerBeans(ServletWebServerApplicationContext.java:266)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.selfInitialize(ServletWebServerApplicationContext.java:240)
	at org.springframework.boot.web.embedded.tomcat.TomcatStarter.onStartup(TomcatStarter.java:52)
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:4426)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:164)
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1203)
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1193)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75)
	at java.base/java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:145)
	at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:749)
	at org.apache.catalina.core.StandardHost.startInternal(StandardHost.java:772)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:164)
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1203)
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1193)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75)
	at java.base/java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:145)
	at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:749)
	at org.apache.catalina.core.StandardEngine.startInternal(StandardEngine.java:203)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:164)
	at org.apache.catalina.core.StandardService.startInternal(StandardService.java:415)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:164)
	at org.apache.catalina.core.StandardServer.startInternal(StandardServer.java:870)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:164)
	at org.apache.catalina.startup.Tomcat.start(Tomcat.java:437)
	at org.springframework.boot.web.embedded.tomcat.TomcatWebServer.initialize(TomcatWebServer.java:128)
	at org.springframework.boot.web.embedded.tomcat.TomcatWebServer.<init>(TomcatWebServer.java:107)
	at org.springframework.boot.web.embedded.tomcat.TomcatServletWebServerFactory.getTomcatWebServer(TomcatServletWebServerFactory.java:516)
	at org.springframework.boot.web.embedded.tomcat.TomcatServletWebServerFactory.getWebServer(TomcatServletWebServerFactory.java:222)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.createWebServer(ServletWebServerApplicationContext.java:188)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.onRefresh(ServletWebServerApplicationContext.java:162)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:621)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:146)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:752)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:439)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:318)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1361)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1350)
	at com.example.demo.WebhealthappApplication.main(WebhealthappApplication.java:14)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.springframework.boot.devtools.restart.RestartLauncher.run(RestartLauncher.java:50)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: La conexión no fue cerrada por la aplicación y esta siendo limpiada (garbage collected)
The connection was not closed by the application and is garbage collected [90018-232]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:690)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 92 more
2024-12-16 23:57:30.415696+01:00 jdbc[25]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabla "USUSARIO" no encontrada
Table "USUSARIO" not found; SQL statement:
SELECT * FROM ususario [42102-232]
2024-12-16 23:57:32.887935+01:00 jdbc[25]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabla "USUSARIOS" no encontrada
Table "USUSARIOS" not found; SQL statement:
SELECT * FROM ususarios [42102-232]
2024-12-16 23:57:38.112578+01:00 jdbc[25]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabla "USUARIOS" no encontrada
Table "USUARIOS" not found; SQL statement:
SELECT * FROM usuarios [42102-232]
2024-12-17 00:01:47.639133+01:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabla "USUSARIO" no encontrada
Table "USUSARIO" not found; SQL statement:
SELECT * FROM ususario [42102-232]
